src/main/java/org/apache/commons/configuration2/interpol/ConfigurationInterpolator.java

```diff
@@ -322,14 +322,14 @@ public boolean isEnableSubstitutionInVariables() {
 
     /**
      * Checks whether a value to be interpolated seems to be a single variable. In this case, it is resolved directly
-     * without using the {@code StringSubstitutor}. Note that it is okay if this method returns a false positive: In this
-     * case, resolving is going to fail, and standard mechanism is used.
+     * without using the {@code StringSubstitutor}.
      *
      * @param strValue the value to be interpolated
      * @return a flag whether this value seems to be a single variable
      */
     private boolean looksLikeSingleVariable(final String strValue) {
-        return strValue.startsWith(VAR_START) && strValue.endsWith(VAR_END);
```

Comment:
Is the check considered strict now? If so, then the method name is no longer appropriate.

Reply:
Yes, it is strict. I've updated the name and the docs.